src/zc/form/browser/tests.py

```diff
@@ -163,6 +164,7 @@ def pageSetUp(test):
 
 def test_suite():
     suite = unittest.makeSuite(TestUnionWidget)
+    suite.addTest(unittest.makeSuite(TestInterfaces))
```

Comment:
I like `unittest.defaultTestLoader.loadTestsFromName(__name__)` -- that way you can add new TestCase subclasses and not worry you'll forget to mention them in the `test_suite()`.

I usually do some variation on

```python
def test_suite():
    return unittest.TestSuite([
        unittest.defaultTestLoader.loadTestsFromName(__name__),
        doctest.DocTestSuite(...),
        doctest.DocFileSuite(...),
    ])
```

Reply:
That is a goof idea, thanks.